demo/src/app/components/+rating/demos/select-on-enter/select-on-enter.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-select-on-enter',
+  templateUrl: './select-on-enter.html'
+})
+export class DemoRatingSelectOnEnterComponent {
+  max = 10;
+  rate = 7;
+  isReadonly = false;
+
+  onKeyup(event) {
+    if (event.keyCode === 13) {
+      this.isReadonly = !this.isReadonly;
```

Comment:
I might be wrong but it seems like in this case user will be able to change the value again just by pressing `Enter` again